cmd/jag/commands/firmware.go

```diff
@@ -111,3 +112,59 @@ func FirmwareUpdateCmd() *cobra.Command {
 	addFirmwareFlashFlags(cmd, "", "new name of the device, if given")
 	return cmd
 }
+
+func FirmwareExtractCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "extract [envelope]",
+		Short: "Build the firmware image of a Jaguar device",
```

Comment:
Not sure if we typically end the short help with dots.

```suggestion
		Short: "Build the firmware image of a Jaguar device.",
```

Reply:
We do in Toit, but not in Go...